### PR TITLE
Routing and login page

### PR DESCRIPTION
**Pull request by Maianki**
### Files to be reviewed
- src/App.js
- src/components/Header/Header.jsx
- src/components/Navbar/Navbar.jsx
- src/index.js
- src/pages/404/PageNotFound.jsx
- src/pages/404/pageNotFound.css
- src/pages/Login/Login.jsx

### Files not to be reviewed
- src/backend/controllers/AuthController.js
- src/backend/controllers/CartController.js
- src/backend/controllers/CategoryController.js
- src/backend/controllers/ProductController.js
- src/backend/controllers/WishlistController.js
- src/backend/db/categories.js
- src/backend/db/products.js
- src/backend/db/users.js
- src/backend/utils/authUtils.js
- src/pages/Login/login.css
- src/server.js
- src/pages/index.js

#### Thanks for reviewing my code

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,11 +1,22 @@
...
-      <LandingPage/>
+      <Routes>
+        <Route path="/" element={<LandingPage/>}></Route>
+        <Route path="/shopNow" element={<>shop Now</>}></Route>
+        <Route path="/login" element={<Login/>}></Route>
+        <Route path="/wishlist" element={<>Wishlist page</>}></Route>
+        <Route path="/cart" element={<>Cart page</>}></Route>
+        <Route path="/mockman" element={<Mockman/>}></Route>
+        <Route path="*" element={<PageNotFound/>}></Route>
+      </Routes>
```
Comment (anik31) [inline src/App.js:19]: appreciation - good that you have added path for unknown routes

File: src/pages/404/PageNotFound.jsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+import "./pageNotFound.css"
+import pageNotFound from "../../assets/images/404.svg"
+export function PageNotFound() {
+  return (
+    <div className='show-404'>
+        <h1>You took a wrong turn</h1>
+        <img className="show-not-found responsive-img" src={pageNotFound} alt="page not found"></img>
+    </div>
+  )
+}
```
Comment (anik31) [inline src/pages/404/PageNotFound.jsx:11]: appreciation - it's good that you have implemented the 404 page

**Review by anik31 — APPROVED** [review]
Nice work 💯 Everything looks great

File: src/App.js [hunk 3]
```diff
@@ -1,11 +1,22 @@
...
     <div className="App">
-      <LandingPage/>
+      <Routes>
+        <Route path="/" element={<LandingPage/>}></Route>
+        <Route path="/shopNow" element={<>shop Now</>}></Route>
+        <Route path="/login" element={<Login/>}></Route>
+        <Route path="/wishlist" element={<>Wishlist page</>}></Route>
+        <Route path="/cart" element={<>Cart page</>}></Route>
+        <Route path="/mockman" element={<Mockman/>}></Route>
+        <Route path="*" element={<PageNotFound/>}></Route>
```
Comment (sainath7878) [inline src/App.js:18]: Does * mean if there is any other path other than the listed paths then, show PageNotFound component?

File: src/components/Header/Header.jsx [hunk 4]
```diff
@@ -12,7 +13,7 @@ export function Header() {
         <div className="hero-img-overlay flex-column">
             <h2 className="hero-heading">Your ultimate stop to buy all</h2>
             <h2 className="hero-heading"> badminton accessories</h2>
-            <button className="btn btn-outline-primary hero-cta text-bold-500 text-center" href="/">Shop Now</button>
+            <Link className="btn btn-outline-primary hero-cta text-bold-500 text-center" to="shopNow">Shop Now</Link>
```
Comment (foolhardy21) [inline src/components/Header/Header.jsx:16]: suggestion: you can avoid camelcase in `to` value of `<Link>` as its a url.

**Review by foolhardy21 — APPROVED** [review]
added a suggestion. rest is good.

File: src/components/Header/Header.jsx (hunk 4 above)
Comment (Maianki) [reply src/components/Header/Header.jsx:16]: Thanks for pointing it out will change it.

File: src/App.js (hunk 3 above)
Comment (Maianki) [reply src/App.js:18]: Yes

**Comment by Maianki** [comment]
Thanks for reviewing @anik31 @foolhardy21

File: src/App.js (hunk 3 above)
Comment (sainath7878) [reply src/App.js:18]: Nice. I did not know this!

**Review by sainath7878 — APPROVED** [review]
Everything looks good. Learnt to use * for all other path in Router and a 404 Page for unknown routes. Good work!

File: src/pages/404/PageNotFound.jsx (hunk 2 above)
Comment (Maianki) [reply src/pages/404/PageNotFound.jsx:11]: Thanks @anik31

**Comment by Maianki** [comment]
Thanks, @sainath7878 for reviewing.

**Pull request merged**